src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -41,7 +41,9 @@ internal static class LogMessages
         public const string IDX10211 = "IDX10211: Unable to validate issuer. The 'issuer' parameter is null or whitespace.";
         public const string IDX10212 = "IDX10212: Issuer validation failed. Issuer: '{0}'. Did not match any: validationParameters.ValidIssuers: '{1}' or validationParameters.ConfigurationManager.CurrentConfiguration.Issuer: '{2}'. For more details, see https://aka.ms/IdentityModel/issuer-validation. ";
         public const string IDX10214 = "IDX10214: Audience validation failed. Audiences: '{0}'. Did not match: validationParameters.ValidAudience: '{1}' or validationParameters.ValidAudiences: '{2}'.";
+        public const string IDX10214S = "IDX10214: Audience validation failed.";
```

Comment:
Adding `S` suffix dor "Short"; keeping the error ID in the name the same to match the existing error ID.

Reply:
yes S for short or scrubbed